templates/base.html

```diff
@@ -11,18 +11,21 @@
 
     <meta name="theme-color" content="#191919">
 
-    <meta property="og:image" content="http{% if request.is_secure %}s{% endif %}://{{ request.get_host }}{% static 'images/black.png' %}">
+    <meta property="og:url" content="https://warsztatywww.pl{{ request.path }}" />
+    <meta property="og:locale" content="PL_pl" />
+    <meta property="og:type" content="website" />
+    <meta property="fb:app_id" content="1522345984699744" />
+    <meta property="og:image" content="http{% if request.is_secure %}s{% endif %}://{{ request.get_host }}{% static 'images/fb_image-cut.jpg' %}">
     <meta property="og:description"
-          content="Wakacyjne Warsztaty Wielodyscyplinarne to coroczna impreza organizowana przez studentów,
+          content="WWW to coroczna impreza organizowana przez studentów,
             przeznaczona dla licealistów zainteresowanych matematyką, informatyką, fizyką lub astronomią.">
```

Comment:
We added biology on the main page, how about here?

Reply:
nah